docs/docs/platform/graphql/index.md

```diff
@@ -1,39 +1,57 @@
 ---
 title: 'GraphQL'
-sidebar_position: 3
+sidebar_position: 1
 ---
 
-GraphQL is a query language for APIs. It provides a complete and understandable description of the data in your API.
+Every Nhost app has it's own autogenerated GraphQL API. The GraphQL API is based on the tables and columns in the [Postgres database](/platform/database) and is instanlty available. It's [Hasura GraphQL engine](https://github.com/hasura/graphql-engine) that powers the GraphQL API.
```

Comment:
Two typos here:
- it's own autogenerated... -> its own autogenerated... 
- instanlty -> instantly